### PR TITLE
New version: Google.AndroidStudio version 2021.2

### PR DESCRIPTION
**Pull request by russellbanks**
﻿- [X] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [X] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [X] Does your manifest conform to the [1.1 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.1.0)?

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/65160)

### CODE REVIEW

File: manifests/g/Google/AndroidStudio/2021.2.1.15/Google.AndroidStudio.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=QUSU.5-1-22000-653
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: Google.AndroidStudio
+PackageVersion: 2021.2.1.15
```
Comment (ItzLevvie) [inline manifests/g/Google/AndroidStudio/2021.2.1.15/Google.AndroidStudio.installer.yaml:5]:
```suggestion
PackageVersion: 2021.2
```
This must match Control Panel.

File: manifests/g/Google/AndroidStudio/2021.2.1.15/Google.AndroidStudio.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,34 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=QUSU.5-1-22000-653
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Google.AndroidStudio
+PackageVersion: 2021.2.1.15
```
Comment (ItzLevvie) [inline manifests/g/Google/AndroidStudio/2021.2.1.15/Google.AndroidStudio.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 2021.2
```
This must match Control Panel.

File: manifests/g/Google/AndroidStudio/2021.2.1.15/Google.AndroidStudio.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=QUSU.5-1-22000-653
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+
+PackageIdentifier: Google.AndroidStudio
+PackageVersion: 2021.2.1.15
```
Comment (ItzLevvie) [inline manifests/g/Google/AndroidStudio/2021.2.1.15/Google.AndroidStudio.yaml:5]:
```suggestion
PackageVersion: 2021.2
```
This must match Control Panel.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by jedieaston — DISMISSED** [review]

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
You'll also need to update the package version folder to be `2021.2`

**Comment by russellbanks** [comment]
> You'll also need to update the package version folder to be `2021.2`

Would this mean the previous manifest would need to be deleted, as there is already a folder for 2021.2?

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**